src/ST/Client/FixtureController.cs

```diff
@@ -0,0 +1,170 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using Baseline;
+using Oakton;
+using StoryTeller.Files;
+using StoryTeller.Model;
+using StoryTeller.Model.Persistence.DSL;
+
+namespace ST.Client
+{
+    public interface IFixtureFileObserver : IDisposable
+    {
+        void Changed(string file);
+        void Added(string file);
+        void Deleted(string file);
+    }
+
+    public interface IFixtureFileWatcher : IDisposable
+    {
+        void WriteFiles(Action action);
```

Comment:
This would only matter if we decide to edit and persist the markdown files from the client, right?